### PR TITLE
Adding a new function to get the full model array with no changes

### PR DESCRIPTION
**Pull request by nwgroup**
Sometimes we need the __fullArr to restore all the model when use some filter

### CODE REVIEW

File: framework/source/class/qx/ui/table/model/Filtered.js [hunk 1]
```diff
@@ -105,6 +105,14 @@ qx.Class.define("qx.ui.table.model.Filtered",
     __applyingFilters : null,
     __filterTypes : null,
 
+    /**
+     * Function to get the full array of the filtered model
+     * @return {Array} the full array of model (with no changes)
+    */
+      getFullArray: function ()
+    {
+	return this.__fullArr;
```
Comment (derrell) [inline framework/source/class/qx/ui/table/model/Filtered.js]: Please fix the indentation.

**Review by derrell — CHANGES_REQUESTED** [review]
Once the indentation is fixed, I have no problem with this PR. I recall, however, that this rarely-used class is highly inefficient. While you're in here playing around with it, if you'd like to review it and possibly offer a (separate) PR that makes it more efficient, that would be really useful.

File: framework/source/class/qx/ui/table/model/Filtered.js (hunk 1 above)
Comment (nwgroup) [reply framework/source/class/qx/ui/table/model/Filtered.js]: Hi, we already fix the indentation. We will review it but at the moment the change help us a lot. Thanks!

**Comment by hkollmann** [comment]
Could you please revert the change to .gitignore?

**Review by hkollmann — CHANGES_REQUESTED** [review]
Please revert .gitignore

**Review by derrell — CHANGES_REQUESTED** [review]
The .gitignore changes shouldn't be in this PR.

**Pull request closed without merging**